### ISSUE TITLE
[Models] Invalid values in images and ground truths

### ISSUE THREAD

**Issue by gonzmg88** [spaceml-org/ml4floods#25]
Satellite images (input to the models) have missing values (in Sentinel-2 encoded as 0).
Ground truth mask (output to the models) could also have missing values (also encoded as the 0 class).

Currently ground truths are set to invalid when the input is invalid. `worldfloods_internal/compute_meta_tiff.py`

Options for training the model (how we should deal with invalids in the input/output the loss function):
- Option 1: replace invalid in output with land class (inpainting?)
- Option 2: learn a separate invalid class
- Option 3: Mask out invalids from loss
- Option 4: use uncertainty as a proxy to invalids (`[.333, .333, .333]`)
- Option 5: use a model to inpaint the image (cloud or/and missing removal model)

If more than 1 option is implemented we can have a value in the config.yaml file that selects what's the _invalid policy_ used for training.
